### PR TITLE
Technical/add js clien

### PR DESCRIPTION
**Pull request by psoap**

### CODE REVIEW

File: src/main/javascript/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Registration:
```
Comment (Logrus86) [inline src/main/javascript/README.md]: Could we use the same readme as we have for java client + specific JS info?

File: src/main/javascript/Runner.js [hunk 2]
```diff
@@ -0,0 +1,754 @@
...
+    return {
+        /**
+         * @return next hero action
+         */
+        get: function () {
+            var me = board.getMe();
+            //console.log(me.getX(), me.getY());
+
+            // TODO your code here
+            var dir = Direction.STOP;  // STUB get any random direction except Direction.STOP
```
Comment (Logrus86) [inline src/main/javascript/Runner.js]: Could we accent here that only two Direction options are supported by ExciteBike?

File: src/main/javascript/README.md (hunk 1 above)
Comment (psoap) [reply src/main/javascript/README.md]: yeah, but readme for js need litle changes, in js we dont have these enums BikeType, GameElementType, SpringboardElementType; we have only Elements var

File: src/main/javascript/Runner.js (hunk 2 above)
Comment (psoap) [reply src/main/javascript/Runner.js]: sure

**Pull request merged**